schema.graphql

```diff
@@ -2037,7 +2037,7 @@ type Distribution @entity {
   localId: Int! @index(unique: false)
   asset: Asset! @index(unique: false)
   portfolio: Portfolio!
-  currency: String!
```

Comment:
Is this always an Asset? Or can people put in a bad ID? I forget if the chain validates for the existence of it or not

Reply:
I guess distribution should be safe. I was thinking of offering